### PR TITLE
Profile and About Pages Refreshed

### LINKED ISSUE #2575: Update Profile Menu
Remove redundant content and clean it up

### PR DESCRIPTION
**Pull request by velkymx**
This update resolves #2575 

Changes:

- Profile menu changed to use list group
- Profile menu cleaned up to removed duplicate links
- About menu updated to change report bug to github issue
- About page updated to remove Get Help link
- About menu updated to change report bug to github issue
- Changed text to Profile Settings
- Added View Profile

### CODE REVIEW

**Comment by velkymx** [comment]
image

File: ProcessMaker/Http/Middleware/GenerateMenus.php [hunk 1]
```diff
@@ -153,7 +153,7 @@ public function handle(Request $request, Closure $next)
 
         Menu::make('sidebar_about', function ($menu) {
             $submenu = $menu->add(__('About'));
-            $submenu->add(__('Profile'), [
+            $submenu->add(__('Profile Settings'), [
```
Comment (ryancooley) [inline ProcessMaker/Http/Middleware/GenerateMenus.php]: Recommend we change this to "Edit Profile" to better pair with "View Profile" as well as the breadcrumb navigation.

File: resources/js/components/NavbarProfile.vue [hunk 2]
```diff
@@ -1,47 +1,49 @@
...
+          <li class="list-group-item px-1">
+            <a :href="user_id">
+              <i class="fas fa-user fa-fw fa-lg"></i>
+              {{$t('View Profile')}}
+            </a>
+          </li>
+          <li class="list-group-item px-1">
+            <a href="/profile/edit">
+              <i class="fas fa-user-cog fa-fw fa-lg"></i>
+              {{$t('Profile Settings')}}
```
Comment (ryancooley) [inline resources/js/components/NavbarProfile.vue]: Recommend we change this to "Edit Profile" to better pair with "View Profile" as well as the breadcrumb navigation.

File: ProcessMaker/Http/Middleware/GenerateMenus.php [hunk 3]
```diff
@@ -165,7 +165,7 @@ public function handle(Request $request, Closure $next)
                 'target' => '_blank'
             ]);
             $submenu->add(__('Report an issue'), [
```
Comment (ryancooley) [inline ProcessMaker/Http/Middleware/GenerateMenus.php]: Recommend changing this to "Report Issue" to match the formal tone of the other menu items.

File: resources/views/about/index.blade.php [hunk 4]
```diff
@@ -20,11 +20,9 @@
         <div class="card card-body">
         <img class="about-logo" src="/img/md-blk-logo.png">
         <hr>
-        <div>{{__('ProcessMaker 4')}} Beta 10 Release Candidate 1</div>
+        <div>{{__('ProcessMaker 4')}} Beta 10 Release Candidate 2</div>
         <hr>
-        <a href="https://docs.google.com/forms/d/e/1FAIpQLScnYje8uTACYwp3VxdRoA26OFkbfFs6kuXofqY-QXXsG-h9xA/viewform" target="_blank">{{__('Report an issue')}}  <i class="fas fa-caret-right fa-lg float-right mr-1"></i></a>
-        <hr>
-        <a href="https://github.com/ProcessMaker/bpm" target="_blank">{{__('Get Help')}}  <i class="fas fa-caret-right fa-lg float-right mr-1"></i></a>
+        <a href="https://github.com/ProcessMaker/processmaker/issues/new" target="_blank">{{__('Report an issue')}}  <i class="fas fa-caret-right fa-lg float-right mr-1"></i></a>
```
Comment (ryancooley) [inline resources/views/about/index.blade.php]: Recommend changing this to "Report Issue" to match the formal tone of the other menu items.

**Pull request merged**